changelog.d/20240710_174404_cedric.bosselut_reword_error_message_for_honeytoken.md

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
If you don't want to add a changelog entry, you can just add the label `skip_changelog` to the PR. Otherwise, please add a small sentence describing the changes in the relevant section :)

Reply:
Thanks, I added the label as this change should not be in changelog in my opinion